cypress/integration/seededFlows/publishingFlows/uploadImage.spec.js

```diff
@@ -18,9 +18,9 @@ describe('Upload image', () => {
       });
 
       // Confirm the UI has updated to show the uploaded state
-      cy.findByRole('button', {
-        name: 'Copy Markdown for imageCopy...',
-      }).should('exist');
+      cy.findByRole('form', { name: 'Edit post' })
+        .find('.spec__image-markdown-copy')
+        .should('exist');
```

Comment:
```suggestion
     cy.findByRole('button', {
        name: 'Copy markdown for image',
      }).should('exist');
```
This works fine locally, and lets us keep checking the accessible name of the button is as expected 👍